recipes/eval/paths.patch

```diff
@@ -0,0 +1,112 @@
+--- combine_eval_reports.pl	2003-05-20 11:52:45.000000000 -0500
++++ combine_eval_reports.pl.new	2019-03-31 01:58:52.812255998 -0500
+@@ -1,4 +1,4 @@
+-#!/usr/bin/perl
++#!/opt/anaconda1anaconda2anaconda3/bin/perl
```

Comment:
You'll want to change these to `#!/usr/bin/env perl`.

Reply:
@dpryan79 We may have to revisit this. Technically, it should be the absolute path to the current `perl`, so that conda can, same as for python, pick up that path and rewrite it to point to the perl from the env into which the package is installed. Relevant because the perl include path isn't set AFAIK, so it needs to be the right perl instance for the deps to be found.

Reply:
Doesn't perl change the path in the shebang once it actually installs modules, as python does?